requirements-dev.txt

```diff
@@ -1,4 +1,4 @@
 pytest
 flake8
 autopep8
-multiversx-sdk-wallet==0.7.0
+multiversx-sdk-wallet @ git+https://github.com/multiversx/mx-sdk-py-wallet@remove-isignable
```

Comment:
Will be changed(in the future) to `multiversx-sdk-wallet==0.8.0` when [this PR](https://github.com/multiversx/mx-sdk-py-wallet/pull/25) is merged and a new version is released.